circle.yml

```diff
@@ -12,5 +12,6 @@ dependencies:
 
 test:
   override:
-    - docker run -e "MIX_ENV=test" panacea mix test
+    - mkdir -p $CIRCLE_TEST_REPORTS/exunit
+    - docker run -e "MIX_ENV=test" -e "CI=true" -v $CIRCLE_TEST_REPORTS/exunit:/test-reports panacea /bin/bash -c "mix test; cp _build/test/lib/panacea/test-junit-report.xml /test-reports"
```

Comment:
actually perhaps not, if we're just running one `run` command and not leaving the container up running. for another project, in order to used `cp` to get files into the container without using volumes, I start the container running with just `/bin/bash` and then do whatever `cp`s I need and then `exec` the command which is `mix test` here